test/system/runScenario2.js

```diff
@@ -0,0 +1,19 @@
+const initTraders = require('./initTraders');
+const systemUpdate = require('./systemUpdates');
+
+const runScenario2 = (trader1, trader2) => {
+  return new Promise((resolve) => {
+    trader2.placeOrder('S')
+      .then(() => systemUpdate(trader1, trader2))
+      .then(() => initTraders(trader1, trader2))
+      .then(() => trader2.placeOrder('B'))
```

Comment:
Since scenario 1 will cancel all its orders, scenario 2 will have to set up its trader 1